### DISCUSSION TITLE
I would like an ability to create my own categories for sorting/filtering the long list of torrents

### DISCUSSION THREAD

**Discussion by ANGulchenko** [transmission/transmission#3548] [Ideas]
There are some torrents that I seed for years and I don't really need/want to see them in the main list.
It would be nice to create a particular category for them (like "My torrents"/"Books"/etc) and have the ability to mark torrents as belonging to those categories.
So as a result the main screen will show only current new uncategorized torrents and those infinite-time-always-there-to-be torrents will be shown only if I chose to see the appropriate category.

What do you think?

**Comment by Petrprogs** [comment]
Yeah, good idea. This feature have utorrent.
![image](https://user-images.githubusercontent.com/76732207/182021384-9a0e0f79-3b3a-4792-8c67-9ac2847a0127.png)

**Comment by maxz** [comment]
Labels are already supported in `libtransmission`. You would just need your frontend to support them as well.
